packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -10543,16 +10543,25 @@ export class AdapterClass extends EventEmitter {
         if (!this.#states) {
             return;
         }
+
+        /** Time after which states regularly set by the status report expire in seconds */
+        const reportStatusExpirySec = Math.floor(this._config.system.statisticsInterval / 1_000) + 10;
+
         const id = `system.adapter.${this.namespace}`;
         this.#states.setState(`${id}.alive`, {
             val: true,
             ack: true,
-            expire: Math.floor(this._config.system.statisticsInterval / 1000) + 10,
+            expire: reportStatusExpirySec,
             from: id
         });
         this.outputCount++;
         if (this.connected) {
-            this.#states.setState(`${id}.connected`, { val: true, ack: true, expire: 30, from: id });
```

Comment:
For me it looks like this would have been resetted if someone would have configured the `StatisticsInterval` to a value larger than `30000` or even approx `30000`

But let me know what you think @Apollon77

Reply:
should be in sync ... in fact "noone" changes ever statistic interval ... but we should change it to respect it now that we saw it :-)